extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -519,6 +498,28 @@ export class Bind {
   }
 
   /**
+   * Observes the dynamic children of `element` for mutations, if any,
+   * for rescanning for bindable attributes.
+   * @param {!Element} element
+   */
+  observeElementIfNecessary_(element) {
+    const observeElement = elementToObserve => {
+      this.mutationObserver_.observe(elementToObserve, {childList: true});
+    };
```

Comment:
He means this arrow function can be an instance method. Made the change, though at a readability cost due to new ivar and method.